ports/cpu-features/portfile.cmake

```
@@ -5,26 +5,35 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO google/cpu_features
-    REF b9593c8b395318bb2bc42683a94f962564cc4664 # 0.4.1
-    SHA512 8c12b50741e2979a32b69c788934bee0d00811b7662006c8b493e98d5efeada67ed59460be40c234b2d3bafd85671cb1a1d7c1a6ee535a7fc1cc6ac56a754576
+    REF a8397ba4591237c17d18e4acc091f5f3ebe7391e # 0.6.0
+    SHA512 71a583e8190699d6df3dfa2857886089265cdfbcb916d9828a3611a1d6d23487464d6448b900b49637f015dd7d4e18bb206e0249af0932928f8ced13a081d42b
     HEAD_REF master
+    PATCHES make_list_cpu_features_optional.patch
 )
 
-vcpkg_configure_cmake(
+# If feature "tools" is not specified, disable building/exporting executable targets.
+# This is necessary so that downstream find_package(CpuFeatures) does not fail.
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+    tools BUILD_EXECUTABLE
+)
+
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
+        ${FEATURE_OPTIONS}
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/CpuFeatures TARGET_PATH share/CpuFeatures)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/CpuFeatures TARGET_PATH share/CpuFeatures)
 
 if("tools" IN_LIST FEATURES)
     vcpkg_copy_tools(TOOL_NAMES list_cpu_features)
+    vcpkg_clean_executables_in_bin(FILE_NAMES list_cpu_features)

```

```suggestion
    vcpkg_copy_tools(TOOL_NAMES list_cpu_features AUTO_CLEAN)
```
Seems this function is not needed here, just pass `AUTO_CLEAN` argument in `vcpkg_copy_tools`. Please see https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_clean_executables_in_bin.md#notes